spec/controllers/keywords_controller_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'sidekiq/testing'
```

Comment:
can we go without this @gutakk 🤔

Reply:
Fixed in 8490b86df58f5140863785db94806164967a4ed9